open-liberty/content.md

```diff
@@ -0,0 +1,148 @@
+# Overview
+
+The images in this repository contain Open Liberty. For more information about Open Liberty, see the [Open Liberty Website](https://openliberty.io/) site.
+
+# Images
+
+There are multiple images available in this repository.
+
+The `kernel` image contains the Liberty kernel and can be used as the basis for custom built images that contain only the features required for a specific application. For example, the following Dockerfile starts with this image, copies in the `server.xml` that lists the features required by the application.
+
+```dockerfile
+FROM %%IMAGE%%:kernel
+COPY server.xml /config/
+```
+
+The `microProfile1` image contains the features required to implement Eclipse MicroProfile 1.2. The `webProfile7` image contains the features required for Java EE7 Web Profile compliance. The `javaee7` image adds the features required for Java EE7 Full Platform compliance. The `javaee7` image is also tagged with `latest`.
+
+There are also additional images for different JVM combinations. Currently there are tags for java8 only, but there are two variants one based on IBM Java and Ubuntu and the other based on the IBM small footprint Java which is based on alpine linux. The naming structure for the variants is tag-javaversion-vandor/variant. This leads to webProfile7-java8-ibmsfj as one. At this time the full list of images are:
+
+	kernel
+	kernel-java8-ibm
+	kernel-java8-ibmsfj
+	microProfile1
+	microProfile1-java8-ibm
+	microProfile1-java8-ibmsfj
+	webProfile7
+	webProfile7-java8-ibm
+	webProfile7-java8-ibmsfj
+	javaee7
+	javaee7-java8-ibm
+	javaee7-java8-ibmsfj
```

Comment:
None of this section applies to the current image, right? (since it's only got `kernel` tags?)

Reply:
I'm planning on committing those tags once the kernel ones get accepted. I wasn't sure I could create tags for Dockerfiles that depend on other Dockerfiles in the initial commit for the official images. However I will prepare a commit that removes this until I provide a commit with those tags.